qa/workunits/cephtool/test.sh

```diff
@@ -1137,8 +1137,10 @@ function test_mon_osd()
   expect_false ceph osd find xyz
   expect_false ceph osd find 0.1
   ceph --format plain osd find 1 # falls back to json-pretty
-  ceph osd metadata 1 | grep 'distro'
-  ceph --format plain osd metadata 1 | grep 'distro' # falls back to json-pretty
+  if [ `uname` != FreeBSD ]; then
```

Comment:
the distro info collected by `ceph osd metadata` comes from the output `lsb_release(1)`, which in turn is a Linux specific command. so, maybe we should put something like:
```sh
if [ `uname` == Linux ]; then
 ...
fi
```